app.py

```diff
@@ -0,0 +1,41 @@
+from flask import Flask,request
+from flask_restful import Resource, Api
+from config import API_Status, API_Status_Message
+import numpy as np
+from class_lib.linear_regression import Predict_Price
+
+
+app = Flask(__name__)
+api = Api(app)
+
+class Boston_Price(Resource):
+    def post(self):
+        data = request.get_json(force=True)
+        if "inputs" in data :
```

Comment:
You should make a separate function for parsing the input. This make make the code cleaner